microshift_getting_started/microshift-architecture.adoc

```diff
@@ -10,7 +10,24 @@ Learn the specifics of {product-title} architecture including design intent, how
 
 [id="microshift-architectural-design_{context}"]
 == Architectural design
-{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
+{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. {product-title} is tasked with only the following Kubernetes cluster services:
```

Comment:
If we're rewording this, maybe we can emphasize MicroShift as a platform for deploying apps, but not so much for developing them (in the way that OCP is)?

I'm not sure about the word "services" here. That can have a very specific meaning that doesn't apply in the example. Is a generic term like "features" OK? Or "capabilities"?

```suggestion
{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to deploying applications in edge computing environments. {product-title} is tasked with only the following Kubernetes cluster services:
```

Reply:
Yep, took another stab at fleshing that out.